<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>265</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>468</width>
    <height>265</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="tcpStartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start TCP server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpDataLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>data</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="tcpStopButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop TCP server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpPortLineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="udpStartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start sending</string>
    </property>
   </widget>
   <widget class="QPushButton" name="udpStopButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop sending</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="udpDataLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>data</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="udpPortLineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="udpIpLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="udpDelayLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>delay, ms</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="udpData2LineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>more data</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpData2LineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>more data</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpDelayLineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>delay, ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpStatusLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Offline</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="udpStatusLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Offline</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>status</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
